<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWindow</class>
 <widget class="QWidget" name="SettingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QProgressBar" name="micVolumeBar">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>450</y>
     <width>241</width>
     <height>27</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="format">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="testVideoButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>325</y>
     <width>99</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>视频设备测试</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Courier</family>
    </font>
   </property>
   <property name="text">
    <string>传输速率</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="fpsSpinBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>44</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>15</number>
   </property>
   <property name="maximum">
    <number>60</number>
   </property>
   <property name="value">
    <number>30</number>
   </property>
  </widget>
  <widget class="QComboBox" name="audiodeviceComboBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>201</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Courier</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>帧率</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>150</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Courier</family>
    </font>
   </property>
   <property name="text">
    <string>分辨率</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>150</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Courier</family>
    </font>
   </property>
   <property name="text">
    <string>音频设备</string>
   </property>
  </widget>
  <widget class="QComboBox" name="bitrateComboBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Courier</family>
    </font>
   </property>
   <item>
    <property name="text">
     <string>自动</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2048Kbit/s</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1024Kbit/s</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>512kbit/s</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>256Kbit/s</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="videoQualityComboBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Courier</family>
    </font>
   </property>
   <item>
    <property name="text">
     <string>1920×1080</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1280×768</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1440x900</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>600x400</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="portSpinBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>76</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Courier</family>
    </font>
   </property>
   <property name="minimum">
    <number>1025</number>
   </property>
   <property name="maximum">
    <number>65520</number>
   </property>
   <property name="value">
    <number>30000</number>
   </property>
  </widget>
  <widget class="QComboBox" name="videoDeviceComboBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>201</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Courier</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Courier</family>
    </font>
   </property>
   <property name="text">
    <string>本地端口</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>150</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Courier</family>
    </font>
   </property>
   <property name="text">
    <string>视频设备</string>
   </property>
  </widget>
  <widget class="QPushButton" name="testAudioButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>99</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>音频设备测试</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>241</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
